packages/cli/src/utils/cli.ts

```diff
@@ -175,3 +177,9 @@ export async function binaryPrompt(promptMessage: string, defaultToNo?: boolean)
 export function getCurrentTimestamp() {
   return Math.floor(Date.now() / 1000)
 }
+
+export function humanizeRequirements(requirements: LockedGoldRequirements) {
+  const requiredCelo = formatEther(requirements.value.toFixed())
+  const requiredDays = requirements.duration.toNumber() / (60 * 60 * 24)
```

Comment:
unlikely to be the case but what if that's a float? Would be weird to have to wait 2.3 days

Reply:
rewriting to use the humanize duration lib